nilearn/plotting/_utils.py

```diff
@@ -92,68 +92,97 @@ def get_cbar_ticks(vmin, vmax, offset, n_ticks=5):
 
 
 def get_colorbar_and_data_ranges(
```

Comment:
In this function, instead of raising error for inconsistent values, it is also possible to warn the user and change them.